class.jetpack.php

```diff
@@ -1034,6 +1035,35 @@ public function check_twitter_tags() {
 			require_once JETPACK__PLUGIN_DIR . 'class.jetpack-twitter-cards.php';
 		}
 	}
+	
+	
+	
+	/**
+	 * Allows plugins to self report for our security dashboard.  OPTION #1
+	 *
+	 * @return null
+	 */
+	public function perform_security_reporting() {
+
+		$security_report = array(
+			'botnet' 		=> array(), //Values: 'plugin'=>'Plugin Name', 'blocked'=>(int)(optional)
+			'backup' 		=> array(), //Values: 'plugin'=>'Plugin Name', 'last'=>(datetime)(optional), 'next'=>(datetime)(optional)
+			'file_scanning' => array(), //Values: 'plugin'=>'Plugin Name', 'last'=>(datetime)(optional), 'next'=>(datetime)(optional)
```

Comment:
These lines seem to be using tabs, not spaces to align.  Indent with tabs, align with spaces.